modules/veteran/app/models/veteran/user.rb

```diff
@@ -35,8 +30,11 @@ def set_attributes(attributes_hash)
       attributes_hash.each { |key, value| send("#{key}=", value) if respond_to?("#{key}=") }
     end
 
-    def auth_headers
-      @auth_headers ||= EVSS::AuthHeaders.new(@user).to_h
+    def bgs_service
+      @bgs_service ||= BGS::Services.new(
+        external_uid: Settings.bgs.external_uid,
```

Comment:
These need to be the ICN and email of the veteran. I think they said we could just pass PID for both also

Reply:
Updated to include the PID.